src/OperatingSystemApp/toolbox.js

```diff
@@ -10,10 +10,16 @@ var OSToolbox = GObject.registerClass(class OSToolbox extends Toolbox {
         this._controlPanel = new OSControlPanel({visible: true});
         this.add(this._controlPanel);
         this.show_all();
+
+        this.connect('reset', () => {
+            this._controlPanel.reset();
+        });
     }
 
     bindWindow(win) {
-        void (this, win);
+        win.lockscreen.key = 'item.key.OperatingSystemApp.1';
```

Comment:
I'm changing the quests to give out this key name, so let's make sure this name doesn't change.